crates/revm/src/processor.rs

```diff
@@ -613,8 +634,10 @@ mod tests {
     }
 
     impl StateRootProvider for StateProviderTest {
-        fn state_root(&self, _bundle_state: &BundleStateWithReceipts) -> RethResult<B256> {
-            todo!()
+        fn state_root(&self, bundle_state: &BundleStateWithReceipts) -> RethResult<B256> {
+            bundle_state
+                .state_root_slow(&database::Database::tx(&self.db)?)
+                .map_err(|err| RethError::Database(err.into()))
         }
```

Comment:
the database & bundle state test are not consistent w/ the state of the `StateProviderTest`. i would simply recommend swapping `todo!()` with `unimplemented!("state root computation is not supported")` since we don't intend to test state root here

Reply:
@rkrasiuk Ok makes sense just changed the implementation